### PR TITLE
fix bato not loading chapter images

### PR DESCRIPTION
**Pull request by ChillDryad**
Fixes Bato having problems loading images.

### CODE REVIEW

**Comment by ChillDryad** [comment]
hold off on merging this one, bato pushed a new update & now doesn't feel like working again.

**Comment by ChillDryad** [comment]
Ready to go now, `imgAccListString` seems to no longer be provided.

File: src/runners/bato/parser.ts [hunk 1]
```diff
@@ -255,14 +255,7 @@ export class Parser {
     const imgAccListString = AES.decrypt(batoWord, evaluatedPass).toString(
       enc.Utf8
     );
-
-    if (!imgAccListString) throw new Error("Invalid Acc List String");
-
     const imgAccList: string[] = JSON.parse(imgAccListString);
```
Comment (Seyden) [inline src/runners/bato/parser.ts:258]: what happens if you parse a empty object or string?

File: src/runners/bato/parser.ts [hunk 2]
```diff
@@ -255,14 +255,7 @@ export class Parser {
...
-
-    if (!Array.isArray(imgAccList) || imgAccList.length == 0)
-      throw new Error("Acc List Is Invalid");
-
     const urls = imgHttpList.map((v, i) => `${v}?${imgAccList[i]}`);
```
Comment (Seyden) [inline src/runners/bato/parser.ts:259]: Same goes here, don't you get a null ref exception or out of bounds exception if the collection doesn't exist and you try to access it?

File: src/runners/bato/parser.ts (hunk 1 above)
Comment (ChillDryad) [reply src/runners/bato/parser.ts:258]: On my end this is always `[ ]`. Just an empty array, JSON.parse _really_ doesn't care much about what you give it.

File: src/runners/bato/parser.ts (hunk 2 above)
Comment (ChillDryad) [reply src/runners/bato/parser.ts:259]: `imgAccList` still exists, I just don't care if it has any content to it.

**Pull request merged**